packages/core/test/Form.test.jsx

```diff
@@ -209,7 +209,6 @@ describeRepeated('Form common', (createFormComponent) => {
 
     it('should work with oneOf', function () {
       const schema = {
-        $schema: 'http://json-schema.org/draft-06/schema#',
```

Comment:
This caused compilation errors that broke the `isValid()` checks but didn't affect the old `validateFormData()` checks because that handled the error differently